packages/ncids-css/package.json

```diff
@@ -119,7 +118,7 @@
 		]
 	},
 	"uswdsExcludeComments": {
-		"//": "usa-content, usa-dark-background, usa-display, usa-intro, usa-link, usa-paragraph, usa-prose all required for uswds-typography.",
+		"//": "usa-content, usa-dark-background, usa-display, usa-intro, usa-link, usa-paragraph, usa-step-indicator, usa-prose all required for uswds-typography.",
```

Comment:
change this back too - step indicator shouldn't be required for uswds-typography to work